package.json

```
@@ -4,9 +4,12 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "dev": "nuxt-ts",
-    "build": "nuxt-ts build",
-    "start": "nuxt-ts start",
+    "dev": "NUXT_SSR=true nuxt-ts",
+    "dev:static": "nuxt-ts",
+    "build": "NUXT_SSR=true nuxt-ts build",
+    "build:static": "nuxt-ts build",
+    "start": "NUXT_SSR=true nuxt-ts start",
+    "start:static": "nuxt-ts start",
```

```suggestion
    "dev": "NUXT_SSR=1 nuxt-ts",
    "dev:static": "nuxt-ts",
    "build": "NUXT_SSR=1 nuxt-ts build",
    "build:static": "nuxt-ts build",
    "start": "NUXT_SSR=1 nuxt-ts start",
    "start:static": "nuxt-ts start",
```
Usually env variables are set to 1